test/e2e/server/index.ts

```diff
@@ -39,6 +40,27 @@ export interface TestServerEvent<T = unknown> {
   method: 'envelope' | 'minidump' | 'store';
 }
 
+function stream2buffer(stream: Readable): Promise<Buffer> {
+  return new Promise((resolve, reject) => {
+    const buf: Buffer[] = [];
+    stream.on('data', (chunk) => buf.push(chunk));
+    stream.on('end', () => resolve(Buffer.concat(buf)));
+    stream.on('error', (err) => reject(err));
+  });
+}
+
+async function getRequestBody(
+  ctx: Koa.ParameterizedContext<any, Router.IRouterParamContext<any, any>, any>,
```

Comment:
l: Is there any reason we're providing `any` as generics here or could we just use the default `Koa.ParameterizedContext` without explicitly passing generics?